### ISSUE TITLE
How to set Adam learning rate?

### ISSUE THREAD

**Issue by aschoenebeck** [SciSharp/TensorFlow.NET#844]
This throws an exception:

```cs
var adam = keras.optimizers.Adam();
adam.lr = 0.0001f;
```
